components/resource_metering/src/cpu/recorder/mod.rs

```diff
@@ -28,20 +28,20 @@ pub struct RecorderHandle {
 pub struct RecorderHandleInner {
     join_handle: JoinHandle<()>,
     pause: Arc<AtomicBool>,
-    precision_seconds: Arc<AtomicU64>,
```

Comment:
I guess you will find the change to be less (and better) if you accept `ReadableDuration` in config and accept milliseconds in `ResourceMetering/*`, which can be fit in an atomic variable instead of must use a Mutex.

Reply:
addressed